boards/arm/stm32_min_dev/stm32_min_dev_defconfig

```diff
@@ -0,0 +1,47 @@
+CONFIG_ARM=y
+CONFIG_BOARD_STM32_MIN_DEV=y
+CONFIG_SOC_FAMILY_STM32=y
+CONFIG_SOC_SERIES_STM32F1X=y
+CONFIG_SOC_STM32F103X8=y
+CONFIG_CORTEX_M_SYSTICK=y
+
+# 72MHz system clock
+CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=72000000
+
+# enable uart driver
+CONFIG_SERIAL=y
+CONFIG_UART_STM32=y
+
+# enable USART1
+CONFIG_UART_STM32_PORT_1=y
+
+# enable console on this port by default
+CONFIG_CONSOLE=y
+CONFIG_UART_CONSOLE=y
+CONFIG_UART_CONSOLE_ON_DEV_NAME="UART_1"
+
+# enable pinmux
+CONFIG_PINMUX=y
+CONFIG_PINMUX_STM32=y
+
+# enable GPIO ports A, B
+CONFIG_GPIO=y
+CONFIG_GPIO_STM32=y
+CONFIG_GPIO_STM32_PORTA=y
+CONFIG_GPIO_STM32_PORTB=y
+CONFIG_GPIO_STM32_PORTC=n
+CONFIG_GPIO_STM32_PORTD=n
+
+# clock configuration
+CONFIG_CLOCK_CONTROL=y
+CONFIG_CLOCK_CONTROL_STM32F10X=y
+CONFIG_CLOCK_STM32F10X_SYSCLK_SRC_PLL=y
+
+# use HSE as PLL input
+CONFIG_CLOCK_STM32F10X_PLL_SRC_HSE=y
+CONFIG_CLOCK_STM32F10X_PLL_MULTIPLIER=9
+CONFIG_CLOCK_STM32F10X_AHB_PRESCALER=0
+
+# APB1 clock must not exceed 36MHz limit
+CONFIG_CLOCK_STM32F10X_APB1_PRESCALER=2
+CONFIG_CLOCK_STM32F10X_APB2_PRESCALER=0
```

Comment:
Update with new version of Clock control  (cf other STM32 boards settings)
CONFIG_CLOCK_STM32F10X_... don't exist anymore today
Btw, you'll need to provide external clock speed (it is kind of useful information we'd like to be able to get in board doc)